src/App/App.js

```diff
@@ -9,34 +11,72 @@ class App extends Component{
     super()
 
     this.state = {
-      userData: {},
-      displayLoginForm : false
+      userData:  {
+        id: null,
+        name: '',
+        email: ''
+      },
     }
   }
 
   buttonHandling = (event) => {
     if(event.target.innerHTML === 'Log in') {
-      this.setState({ displayLoginForm : true })
+      console.log('button')
     } 
 }
```

Comment:
This is not doing anything and forgot to remove it

Reply:
Haha gotcha! Thanks for the comment.